pkgs/servers/http/lighttpd/default.nix

```diff
@@ -45,6 +47,7 @@ stdenv.mkDerivation rec {
                 ++ stdenv.lib.optional enableExtendedAttrs "--with-attr";
 
   preConfigure = ''
+    export PATH=$PATH:${pcre.dev}/bin
```

Comment:
I think we can avoid this by putting pcre.dev in nativeBuildInputs

Reply:
it should also be `dev` by default? Just put `pcre` in both `buildInputs` and `nativeBuildInputs`. No `PATH` by hand, and no `.dev` by hand.

Reply:
Thing is `pcre.dev` provides `pcre-config`, which is bash script that's needed to configure script, and we need target version. If i put it to nativeBuildInputs it will gladly cofigure, but at compile time it will try and fail to link it against host pcre library.

Reply:
Oh it's `*-config` hell. See https://github.com/NixOS/nixpkgs/issues/51176 .

OK what you did is fine, but add a comment explaining that. Also, it's build and host platform, not host and target platform for this.

Reply:
Yeah nevermind. What's being used now is reasonable.